docs/deprecated.md

```diff
@@ -14,17 +14,15 @@ keywords: "docker, documentation, about, technology, deprecate"
      will be rejected.
 -->
 
-# Deprecated Engine Features
-
 The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
 ### Asynchronous `service create` and `service update`
 
-**Deprecated In Release: v17.05.0**
+## Deprecated In Release: v17.05.0
```

Comment:
Also, use sentence case "Deprecated in release <foo>" (don't capitalize each word)

Reply:
Ok, I will do that as well.